app/Http/Controllers/Service/Admin/MarketsController.php

```diff
@@ -25,6 +25,11 @@ class MarketsController extends Controller
     public function index()
     {
         $markets = Market::with(['sponsor', 'traders'])->get();
+        // TODO : efficiency
```

Comment:
create a somewhat default sort on the markets table

Reply:
I think you can use the groupBy method and you may pass multiple arguments to group by multiple columns
`$users = DB::table('users')->groupBy('first_name', 'status')->get();` it also takes a param for the order - asc or desc

Reply:
At the moment i'm happy to add this to a card I shall make shortly